src/QuestionHtml.tsx

```diff
@@ -92,17 +92,21 @@ class MyWebView extends React.Component<any, any> {
 
     return (
       <WebView
+        {...this.props}
+        // androidHardwareAccelerationDisabled={
+        //   !!'https://github.com/react-native-webview/react-native-webview/issues/811'
+        // }
+        automaticallyAdjustContentInsets={true}
         injectedJavaScript={injectedScript}
-        scrollEnabled={this.props.scrollEnabled || false}
-        onNavigationStateChange={this._onNavigationStateChange}
-        onMessage={this._onMessage}
         javaScriptEnabled={true}
-        automaticallyAdjustContentInsets={true}
-        {...this.props}
-        style={[styles.container, this.props.style, { height }]}
-        androidHardwareAccelerationDisabled={
-          !!'https://github.com/react-native-webview/react-native-webview/issues/811'
-        }
```

Comment:
sorted these, removed `androidHardwareAccelerationDisabled` and ran `yarn build`